.github/workflows/node.js.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
I think removing 10 & 12 compatibility is reasonable. ppl who didn't upgrade node for two years probably don't do dependency updates anyway.